src/core/sys/darwin/sys/attr.d

```diff
@@ -0,0 +1,305 @@
+/**
+ * D header file for Darwin
+ */
```

Comment:
Needs proper documentation, and changelog entry

Reply:
It has an associated issue https://issues.dlang.org/show_bug.cgi?id=20787 although the D bot hasn't linked it yet for whatever reason.

Reply:
Added a link to the macOS 10.9 sys/attr.h and put comments next to constants that were added since then. Since public documentation isn't generated for `core.sys.*` I think this is as much as is needed.